packages/block-editor/src/components/global-styles/color-panel.js

```diff
@@ -208,6 +209,7 @@ function ColorPanelDropdown( {
 } ) {
 	const currentTab = tabs.find( ( tab ) => tab.userValue !== undefined );
 	const { key: firstTabKey, ...firstTab } = tabs[ 0 ] ?? {};
+	const colorGradientDropdownButtonRef = useRef();
```

Comment:
Reac 19 will [start complaining](https://react.dev/blog/2024/04/25/react-19-upgrade-guide#useref-requires-argument) about refs without a default value. Feel free to go with a `undefined`:

```suggestion
	const colorGradientDropdownButtonRef = useRef( undefined );
```

Reply:
Perfect, thanks for the heads up.